### PR TITLE
chore: modified server schema version

### PR DESCRIPTION
**Pull request by sondermanish**
Fixes: https://github.com/appsmithorg/appsmith/issues/28364

### CODE REVIEW

File: app/server/appsmith-server/src/main/java/com/appsmith/server/jslibs/imports/CustomJSLibImportableServiceCEImpl.java [hunk 1]
```diff
@@ -65,9 +65,17 @@ public Mono<Void> importEntities(
      * Read More: https://github.com/appsmithorg/appsmith/pull/28012
      *
      * @param customJSLibList
+     * @param applicationJson
      */
-    private void ensureXmlParserPresenceInCustomJsLibList(List<CustomJSLib> customJSLibList) {
+    private void ensureXmlParserPresenceInCustomJsLibList(
+            List<CustomJSLib> customJSLibList, ApplicationJson applicationJson) {
+        // this is to ensure that newer applications(server schema > 6) does not get xml parser when imported.
+        if (applicationJson.getServerSchemaVersion() > 6) {
```
Comment (nidhi-nair) [inline app/server/appsmith-server/src/main/java/com/appsmith/server/jslibs/imports/CustomJSLibImportableServiceCEImpl.java:73]: If this is meant for imported JSON files of older versions, please use the `JsonSchemaMigration`

**Review by nidhi-nair — APPROVED** [review]

**Comment by ramsaptami** [comment]
lgtm cc: @arunvjn

**Pull request merged**